### DISCUSSION TITLE
v3.3.0

### DISCUSSION THREAD

**Discussion by marcelklehr** [nextcloud/recognize#535] [Announcements]
### New
 - Implement recognize:remove-legacy-tags command
 - Implement GPU mode
 - Implement recognize:classify command

<hr /><em>This discussion was created from the release <a href='https://github.com/nextcloud/recognize/releases/tag/v3.3.0'>v3.3.0</a>.</em>

**Comment by marcelklehr** [comment]
You can use the following Dockerfile to run in GPU mode: https://github.com/bugsyb/recognize_docker thanks to @bugsyb

**Comment by GrillGuth** [comment]
Can you please explain the difference between the recognize:classify and recognize:recrawl command?
Thanks :)

**Reply by marcelklehr** [reply]
You can use recrawl to trigger a complete rerun using background jobs (the command will thus exit after a second). classify will do a complete run while the command is running, regardless of the state of the background jobs
